app/components/home/mailing_list_component.html.erb

```diff
@@ -0,0 +1,30 @@
+<section class="row">
+  <div class="home-mailing-list">
+    <div class="col col-full-content">
+      <div>
+        <h2><i class="icon icon-arrow-black icon--on-yellow"></i> Sign up for our emails</h2>
+        <p>
+          Everything you need to know to start a career in teaching sent straight into your inbox.
+          Tailored to your own situation, you get all the latest information as well as advice and support.
+        </p>
+
+        <p>
+          <small>
+            Your details are protected under the terms of our <%= link_to("privacy notice", privacy_policy_path(id: privacy_policy.id), { class: "link--black", target: :blank }) %>.<br>
```

Comment:
@philspray @EllieNodder I'm not that convinced by using a <br> here - I think we should probably either start a new <p> or merge into one proper paragraph?